src/rttest.cpp

```diff
@@ -545,7 +548,21 @@ int Rttest::lock_and_prefault_dynamic()
   size_t encountered_majflts = 1;
   // prefault until you see no more pagefaults
   while (encountered_minflts > 0 || encountered_majflts > 0) {
-    char * ptr = static_cast<char *>(calloc(64 * page_size, sizeof(char)));
+    //char * ptr = static_cast<char *>(calloc(64 * page_size, sizeof(char)));
+    char * ptr;
+    try {
+      ptr = new char[64 * page_size];
+    } catch (std::bad_alloc & e) {
```

Comment:
Since the block below does not use `e` it will likely be an *unused* warning. Should the exception message be appended to the message below?